gator/checks/check_CountSingleLineComments.py

```diff
@@ -0,0 +1,105 @@
+"""Check that a file in a directory contains a specified number of single-line comments."""
+
+import argparse
+
+from gator import checkers
+from gator import constants
+from gator import invoke
+
+
+def get_parser():
+    """Get a parser for the arguments provided on the command-line."""
+    # create the parser with the default help formatter
+    # use a new description since this is a stand-alone check
+    parser = argparse.ArgumentParser(
+        prog="CountSingleLineComments",
+        description="Check Provided by GatorGrader: CountSingleLineComments",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    # Required Named Checker Arguments {{{
+
+    required_group = parser.add_argument_group("required checker arguments")
+
+    # FILE: the file
+    # REQUIRED? Yes
+    required_group.add_argument(
+        "--file", type=str, help="file for checking", required=True
```

Comment:
Okay so basically all the `--file` arguments should be expanded to look like `--directory` et al.